brew.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: brew
+  version: 4.1.23
+  epoch: 0
+  description: "The homebrew package manager"
+  copyright:
+    - license: BSD-2
+  dependencies:
+    runtime:
+      - bash
+      - build-base
+      - busybox
+      - coreutils
+      - curl
+      - gcc
+      - git
+      - grep
+      - posix-libc-utils
+      - procps
+      - ruby-3.1
+  # Drop commands/depends/provides as the SCA doesn't work on brew since itself is a package manager
+  options:
+    no-commands: true
+    no-depends: true
+    no-provides: true
+  scriptlets:
+    post-install: |
+      #!/bin/sh
+      echo
+      echo "To use brew for the first time, you will need /home/linuxbrew/.linuxbrew/bin in your PATH:"
+      echo ". /etc/profile.d/brew.sh"
+      echo
+
+environment:
+  contents:
+    packages:
+      - git
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/Homebrew/brew
+      tag: ${{package.version}}
+      destination: ./brew
+  - runs: |
+     set -x
+     mkdir -p ${{targets.destdir}}/home/linuxbrew/.linuxbrew/
+     rm -rf ./brew/.git/
+     mv ./brew/ ${{targets.destdir}}/home/linuxbrew/.linuxbrew/
+     cd ${{targets.destdir}}/home/linuxbrew/.linuxbrew/brew/bin/
+     mkdir -p ${{targets.destdir}}/etc/profile.d/
+     echo "export PATH=\$PATH:/home/linuxbrew/.linuxbrew/brew/bin" > ${{targets.destdir}}/etc/profile.d/brew.sh
+     echo "export HOMEBREW_NO_AUTO_UPDATE=1" >> ${{targets.destdir}}/etc/profile.d/brew.sh
```

Comment:
How will users consume updates to the Homebrew recipes?  Will they be distributed via the APK or...?

Reply:
I've added a WARNING message that gets printed in the post-install, that running 'brew update' will probably not play nicely with 'apk upgrade brew'.
I've actually tested that (apk install brew ; brew update ; apk upgrade brew), and here's what happens... The brew upgrade goes out to github, and git pulls the latest code into /home/linuxbrew/.linuxbrew/brew. Which runs fine. Then, apk upgrade overwrites the code in the git repo there (which, conveniently shows the changes with a git diff). brew -v actually notices this, and adds a "-dirty" on the end of the version which is kind of nice. If the user wants to get back to the brew version from git, it's just git rebase commands. Otherwise, the user is running the new apk version.